app/services/chat_channels/creation.rb

```diff
@@ -0,0 +1,15 @@
+module ChatChannels
+  class Creation
+    attr_accessor :user, :params
```

Comment:
is there a need to access this from the outside or can they just be private `attr_reader`s ?